website/docs/feature-flag-tutorials/use-cases/security-compliance.md

```diff
@@ -0,0 +1,240 @@
+
+-   Tracking change management workflows
+-   Demonstrating organizational compliance
+
+Highly regulated industries like finance and healthcare typically require extensive log preservation, maintaining detailed records for up to seven years. For most enterprise environments, a three-year retention period provides a balance between compliance and operational efficiency. Unleash provides this insight for your teams and security reviewers in a transparent, detailed, and searchable way. You can also track changes using the global [event timeline](/reference/events#event-timeline) to review recent events for up to 48 hours per environment. Event logs also capture API calls that come in from your applications and services.
+
+In the Unleash Admin UI navigation, go to **Event Logs** to see chronological events with metadata and change diffs. You can use the filter or the search bar to narrow down specific events you may be looking for. And not only are your events visible in Unleash, they’re also reportable. You can export your events as a CSV or JSON file to send to analytics tools.
+
+### Integrate SIEM for advanced audit compliance
+
+For more advanced implementations, integrate Unleash event logs directly into broader Security Information and Event Management (SIEM) systems. This setup enables real-time monitoring and creates a holistic view of system interactions across multiple platforms. By mapping Unleash audit data to other enterprise security logs, organizations can detect subtle patterns and potential security anomalies that might go unnoticed in isolated systems. You can send your logs to tools like [Datadog](/reference/integrations/datadog), [Sentry](https://docs.sentry.io/platforms/javascript/guides/nextjs/configuration/integrations/unleash/), Splunk, S3, and Microsoft Sentinel. SIEM integrations take audit logging to the next level to capture more value out of the data that Unleash provides you.
+
+### Leverage access logs for broader auditing
+
+Let’s think back to the importance of user management that we covered earlier. Developers and other stakeholders go through onboarding to use the platform. Authentication protocols and user provisioning ensure these processes are secure, unified, and automated. During this process, access logs keep track of who has access to Unleash for your organization’s audit compliance. You can export these logs to S3 buckets for long-term data storage. This is valuable if you need to preserve data for complying with legal or regulatory compliance, storing critical backups for disaster recovery, and archiving.
+
+Auditing your feature flag system is made simple for traceability and reportability with Unleash’s event logs and access logs. We recommend leveraging these features as data sources for third-party services that make your data a valuable asset for security reviews, meeting compliance standards, and overall risk mitigation.
```

Comment:
In the Google doc there was a comment from Michael about highlighting that access logs can also tell you about API interactions as well. Maybe instead of saying: "access logs keep track of who has access to Unleash for your organization’s audit compliance" we can say something like "access logs keep track of what users and systems accessed Unleash and what actions they performed."

Reply:
Updated to:

"Let’s think back to the importance of user management that we covered earlier. Developers and other stakeholders go through onboarding to use the platform. Authentication protocols and user provisioning ensure these processes are secure, unified, and automated. During this process, access logs keep track of what users and systems accessed Unleash and what actions they performed, including [Unleash API interactions](/reference/api/unleash) from your services and applications."